include/adaflo/navier_stokes.h

```diff
@@ -168,6 +168,19 @@ class NavierStokes : public FlowBaseAlgorithm<dim>
                            const unsigned int                                face,
                            const double                                      density);
 
+  // Set a user defined material law to compute the viscous stress constribution. You have
+  // to pass a lambda function, which computes @return the shear stress rate depending on the
+  // @p velocity_gradient for a given cell @p cell_idx at the quadrature point @p quad_idx.
+  // @p do_tangent=true means that the function is called from the tangent (vmult);
+  // do_tangent=false means that the residual is computed.
```

Comment:
@peterrum I replaced the `do_reinit` flag by `do_tangent`, to get the information whether the residual or vmult is computed. I hope the description makes the interface clear now.